web/src/engine/osk/src/views/oskView.ts

```diff
@@ -790,10 +790,16 @@ export default abstract class OSKView extends EventEmitter<EventMap> implements
     // Instantly resets the OSK container, erasing / delinking the previously-loaded keyboard.
     this._Box.innerHTML = '';
 
+    // Temp-hack:  embedded products prefer their stylesheet, etc linkages without the /osk path component.
+    let subpath = 'osk/';
+    if(this.config.isEmbedded) {
+      subpath = '';
+    }
```

Comment:
If this is a temp hack, can we make sure we track this?

Reply:
I don't see why not.

I was really tempted to alter the mobile apps to expect the stylesheets within the standard subfolder... but there were enough changes already, so I talked myself down from it.  That'd be a pretty easy resolution.

I do wonder if there's a reason we _didn't_ put them in a subfolder within the mobile apps originally, though - and investigation that factor likely merits resolution in the same tracked issue.